cmd/internal/cli/actions_linux.go

```diff
@@ -34,6 +34,10 @@ import (
 
 // TODO: Let's stick this in another file so that that CLI is just CLI
 func execStarter(cobraCmd *cobra.Command, image string, args []string, name string) {
+	if vmStarter(cobraCmd, image, args) {
```

Comment:
Let's remove this code. Leave `execStarter()` function as a function which is solely responsible for running the `starter` binary on linux platform. Likewise, `execVM()` can be responsible for starting the VM.

Reply:
Let me give this a try